SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -69,41 +58,33 @@ class FLocalDeploymentManager
 	FDelegateHandle WorkerConfigDirectoryChangedDelegateHandle;
 	IDirectoryWatcher::FDirectoryChanged WorkerConfigDirectoryChangedDelegate;
 
-	const FString& GetLocalRunningDeploymentID() { return LocalRunningDeploymentID; }
-
 private:
 	void StartUpWorkerConfigDirectoryWatcher();
 	void OnWorkerConfigDirectoryChanged(const TArray<FFileChangeData>& FileChanges);
 
-	bool FinishLocalDeployment(FString LaunchConfig, FString RuntimeVersion, FString LaunchArgs, FString SnapshotName,
-							   FString RuntimeIPToExpose);
+	bool SetupRuntimeFileLogger(const FString& SpatialLogsSubDirectoryName);
 
 	TFuture<bool> AttemptSpatialAuthResult;
 
-	static const int32 ExitCodeSuccess = 0;
-	static const int32 ExitCodeNotRunning = 4;
+	TOptional<FMonitoredProcess> RuntimeProcess = {};
+	TUniquePtr<IFileHandle> RuntimeLogFileHandle;
+	FDateTime RuntimeStartTime;
+
 	static const int32 RequiredRuntimePort = 5301;
+	static const int32 WorkerPort = 8018;
+	static const int32 HTTPPort = 5006;
 
-	// This is the frequency at which check the 'spatial service status' to ensure we have the correct state as the user can change spatial
-	// service outside of the editor.
-	static const int32 RefreshFrequency = 3;
+	const float RuntimeTimeout = 5.0f;
```

Comment:
Is this not static because of the integral-types-only rule? I think it can be static with constexpr from c++11 though (to match the static/const constants above)